daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala

```diff
@@ -50,9 +50,12 @@ object Blinding {
     if (enrichedTx.failedAuthorizations.isEmpty) {
       Right(
         BlindingInfo(
-          enrichedTx.explicitDisclosure,
-          enrichedTx.localImplicitDisclosure,
-          enrichedTx.globalImplicitDisclosure))
+          disclosure = enrichedTx.explicitDisclosure,
+          // NOTE(MH): Local divulgences are a thing of the past and are not
+          // produced by the current implementation anymore.
+          localDivulgence = Map.empty,
```

Comment:
I don't know if we could be more aggressively clean up here and would need some advice on this. If we can clean up more, I'd leave for a separate PR nevertheless.

Reply:
I thinks @gerolf-da (or maybe @stefanobaghino-da) would be better at giving advice on this.

Reply:
I see only four lines that use `localDivulgence`. You can probably safely remove it and keep this PR very small. In any case, I would prefer if you could at least put this so that it's an invariant at construction, rather than a comment here where it's called.

Reply:
Yes, with the removal of `RelativeContractId`, `localDivulgence` lost it's justification to be a thing.

Reply:
[This](https://github.com/digital-asset/daml/pull/6509) looks like it would do it, testing if it doesn't breaking anything.

Reply:
@stefanobaghino-da I'm fine with merging your suggestion in once we've discussed my comment on it.

Reply:
Ran all tests under `//daml-lf` and one conformance test, all good, unfortunately I cannot run the full thing on CI unless I'm trying to merge on `master`.

Reply:
> Yes, with the removal of `RelativeContractId`, `localDivulgence` lost it's justification to be a thing.

@remyhaemmerle-da is crying of joy.

Reply:
Since it's still unclear if we can apply @stefanobaghino-da's suggestion without breaking backward compatibility, we merge this as is. Because of backward compatibility concerns, we can also not address
> In any case, I would prefer if you could at least put this so that it's an invariant at construction, rather than a comment here where it's called.